### ISSUE TITLE
Doesn't accept taps after repositioning "AdaptiveSheet"

### ISSUE THREAD

**Issue by Killer232323** [tachiyomiorg/tachiyomi#9194]
### Steps to reproduce

Appears every time you accidentally swipe 

### Expected behavior

Must accept clicks on elements

### Actual behavior

https://user-images.githubusercontent.com/49369073/224392186-5835982e-7db9-4c3d-a436-43f7594bfa55.mp4

### Crash logs

What the log says when the bar does not accept clicks:

`D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49549071, downTime=49549071 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49549171, downTime=49549071 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49549477, downTime=49549477 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49549568, downTime=49549477 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49549861, downTime=49549861 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49549939, downTime=49549861 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49550166, downTime=49550166 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49550265, downTime=49550166 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49550420, downTime=49550420 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49550520, downTime=49550420 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49550650, downTime=49550650 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49550742, downTime=49550650 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_DOWN, id[0]=0, pointerCount=1, eventTime=49550892, downTime=49550892 } moveCount:0
D/MIUIInput: [MotionEvent] ViewRootImpl { action=ACTION_UP, id[0]=0, pointerCount=1, eventTime=49550975, downTime=49550892 } moveCount:0`

### Tachiyomi version

Current debug version

### Android version

12

### Device

Poco x3, Mi note 10

### Other details

I couldn't post the bug because it asked me to check the box "If this is an issue with an extension, I should be opening an issue in the extensions repository. *"

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Can't reproduce, and those logs seem MIUI-specific.

**Comment by scb261** [comment]
Can reproduce with both MIUI optimization on and off. It doesn't happen consistently (maybe about 60-70% of times), and when it happens you can keep trying to interact with elements and it will eventually work. Crash log for this is empty, but there is something about `libmigui.so` when starting the app, not sure if it's related.
```
--------- beginning of main
03-15 16:01:53.230  3285  3824 E SQLiteLog: (284) automatic index on C(manga_id)
03-15 16:01:53.230  3285  3824 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
03-15 16:01:53.569  3285  3419 E tachiyomi.debu: open libmigui.so failed! dlopen - dlopen failed: library "libmigui.so" not found
App version: 0.14.5-5620 (standard, 47a2d0668, 99, 2023-03-11T21:09Z)
Android version: 12 (SDK 31)
Android build ID: SKQ1.211103.001 test-keys
Device brand: Redmi
Device manufacturer: Xiaomi
Device name: rain
Device model: 220333QNY
Device product name: rain_eea
```

**Comment by AllLife2la** [comment]
Same behavior on r5653 on miui 12.5.1, emui 11.0.0.
Does not happen on Stable 0.14.5

**Comment by arkon** [comment]
Does this still happen in r5666?

**Comment by AllLife2la** [comment]
Still happens r5666

**Comment by scb261** [comment]
Yes, still happens in r5666. After more testing I've figured a bit more. When the sheet expands, there is a bit of delay before it stops expanding (it still slowly expands). And if you start tapping when it still expands, the bug happens. Attaching video, maybe it will help (ignore first ~5 seconds).

https://user-images.githubusercontent.com/65343233/229376625-30c0ba23-1ec4-4fbd-80c1-b4e41fcb9605.mp4

**Comment by seew3l** [comment]
Could it be a MIUI problem? I deactivated the animations and I no longer have that problem

**Comment by arkon** [comment]
Does this still happen on newer preview builds?

**Comment by bapeey** [comment]
No, I think it was fixed in #10023
